axlearn/experiments/text/gpt/fuji.py

```diff
@@ -188,6 +188,7 @@ def get_trainer_kwargs(
                 num_kv_heads=None if version == Version.V1 else 8,
                 rope_theta=rope_theta,
                 flash_attention=flash_attention,
+                remat_offload_dst="pinned_host",
```

Comment:
Add a comment on the observed MFU and step time?

Reply:
Done